homeassistant/components/amcrest/sensor.py

```diff
@@ -68,22 +64,22 @@ def icon(self):
     @property
     def unit_of_measurement(self):
         """Return the units of measurement."""
-        return SENSORS.get(self._sensor_type)[1]
+        return self._unit_of_measurement
 
     def update(self):
         """Get the latest data and updates the state."""
         _LOGGER.debug("Pulling data from %s sensor.", self._name)
 
         if self._sensor_type == 'motion_detector':
-            self._state = self._camera.is_motion_detected
-            self._attrs['Record Mode'] = self._camera.record_mode
+            self._state = self._api.is_motion_detected
```

Comment:
Isn't this a binary state?

Reply:
This sensor has been deprecated and there is already a binary_sensor taking its place (for just that reason. See PR #22703.) The change here actually has nothing to do with that. I've simply renamed `self._camera` to `self._api` throughout because I believe that is clearer.